packages/sdk/src/oracle/services/optimisticOracleV2.ts

```diff
@@ -1,49 +1,28 @@
 // This is almost a direct copy of the OO v1 service, just with updated abis and different contract addresses
-import { optimisticOracle } from "../../clients";
```

Comment:
the was the original v1 client being imported into the v2 service, which was wrong